homepairs/homepairsUI/src/screens/Main/Properties/TenantPropertyScreen/TenantPropertyScreenBase.tsx

```diff
@@ -72,68 +72,94 @@ const styles =  StyleSheet.create({
+    const { propertyState } = props;
+    const { properties, propertyManager } = propertyState;
+    console.log("properties");
+    console.log(properties);
+    console.log("propertyManager");
+    console.log(propertyManager);
 
     /* BEWARE: styles.addBottomMargin doesn't always work, had to add it manually 
         / overlapping styles aen't currently supported by react
         see ref: https://github.com/facebook/react/issues/2231 */
-    function renderContents(){
-        return(
-        <ScrollView style={{flexGrow: 1}}>
-            <View style={styles.addBottomMargin}>
-                <AddressSticker
-                address={address}/>
-                <View style={styles.imageWrapper}>
-                  <View style={styles.imageContainer}>
-                      <Image 
-                      source={defaultProperty}
-                      style={Platform.OS === 'web' ? styles.homepairsPropertiesImageWeb : styles.homepairsPropertiesImage}
-                      resizeMode='cover'/>
-                  </View>
+
+    function renderProperty(pair: [string, Property]) {
+        const [propId, property] = pair;
+        const { address } = property;
+        const apiKey = 'AIzaSyAtsrGDC2Hye4LUh8jFjw71jita84wVckg';
+
+        return (
+            <ScrollView 
+                contentContainerStyle={{}}
+                style={{flexGrow: 1}}>
+                <View style={styles.addBottomMargin}>
+                    <AddressSticker address={address}/>
+                    <View style={styles.imageWrapper}>
+                        <View style={styles.imageContainer}>
+                            <Image 
+                                source={{uri: `https://maps.googleapis.com/maps/api/streetview?size=600x300&location=${address}&pitch=-0.76&key=${apiKey}`}} 
+                                style={Platform.OS === 'web'
+                                ? styles.homePairsPropertiesImageWeb
+                                : styles.homePairsPropertiesImage} 
+                                resizeMode='cover'/>
+                        </View>
+                    </View>
+                    <GeneralHomeInfo property={property} hasEdit={canEditProps} />
+                    <PrimaryContactInfo propertyManager={propertyManager} />
+                    <ServiceRequestCount property={property} />
                 </View>
-                <GeneralHomeInfo property={property} hasEdit={canEditProps} />
-                <PrimaryContactInfo propertyManager={propertyManager}/>
-                <ServiceRequestCount property={property}/>
-            </View>
-        </ScrollView>
+            </ScrollView>
         );
     }
-    
-    return(
-        !(Platform.OS === 'ios') ? 
-        (
-            <View style={styles.container}>
-                <View style={styles.pallet}>
-                    {renderContents()}
+
+    function renderContents() {
+        return (
+            <FlatList
```

Comment:
Why is a Flatlist being used here? Pretty sure these are meant for very long lists of components.

Reply:
Back when we planned tenants, it was a huge back and forth whether theyd get a single property or a list. The final version got changed to a list (yes, even though they only have one now). For the sake of just getting functionality done / not refactoring a bunch for no difference to the user (its a list that will always only contain one, but i guess it opens up potential for past properties). 

It wouldn't have to be a flatlist, but i needed to update the tenant homepage to reflect the pairs for properties, because I think that had caused it to break. I hadn't considered the complexity, I did it more for consistency since the properties we received were technically still a list.

Reply:
Yes, the conversion from List of properties to a Dictionary did cause it to break. However, there are simple fixes to this. The easiest would be to input the following code below in your component:
`const [ propId, property] = Object.entries(properties)[0]; // This converts dictionaries into a list and grabs the first element `
Since Tenant Property is a functional component, the property has been effectively defined for the entire scope of the component.

Other solutions would to be to set the selected property id (in the reducer) to only property provided or to put the parameter into the route and take from that when the component navigates.